### PR TITLE
Adding Patient DSTU2 documentation

### PR DESCRIPTION
**Pull request by cerbs**
Adding documentation for Patient DSTU2

### CODE REVIEW

File: content/dstu2/patient.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+`identifier`                                             | At least one parameter must be populated other than `_count` | [`token`](http://hl7.org/fhir/DSTU2/search.html#token)  | A patient identifier.  Example: `01022228`
+`name`                                                   | At least one parameter must be populated other than `_count` | [`string`](http://hl7.org/fhir/DSTU2/search.html#string)| A portion of either family or given name of the patient. Example: `Peters`
+`telecom`                                                | At least one parameter must be populated other than `_count` | [`token`](http://hl7.org/fhir/DSTU2/search.html#token)  | The value in any kind of telecom details of the patient. Example: `(816) 475-2374`
+[`_count`](http://hl7.org/fhir/DSTU2/search.html#count)| N | [`number`](http://hl7.org/fhir/DSTU2/search.html#number)| The maximum number of results to return. Defaults to `20`.
+
+Note: Parameters (other than `_id`) will currently perform a "fuzzy" search.
+### Response
```
Comment (jfrhong) [inline content/dstu2/patient.md]: I think there needs to be a blank line above. This isn't rendering properly.

**Comment by jfrhong** [comment]
+1

**Comment by davischristi** [comment]
:+1:

**Comment by cerbs** [comment]
squashing

**Pull request merged**